### PR TITLE
Fixed build on windows

### PR DESCRIPTION
**Pull request by XantreDev**
#554

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "mocha": "^10.0.0",
     "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
+    "shx": "^0.3.4",
```
Comment (rschristian) [inline package.json:92]: As we just need `mv`, I think we can find a simpler/smaller package for that.

I can't recall the name but I'm fairly certain there's a tiny (Windows-compatible) `mv` CLI package, sorta like `rimraf`.

Comment (XantreDev) [reply package.json:92]: When I'm done with research, I found only this one. 
I can write a script in JavaScript if you prefer it

Comment (rschristian) [reply package.json:92]: [`move-cli`](https://npm.im/move-cli) perhaps? I guess alternatively we could drop `rimraf` for `shx`?

Comment (XantreDev) [reply package.json:91]: I've replaced rimraf with `shx`

**Review by rschristian — APPROVED** [review]

**Pull request merged**